File: src/Python/examples/adapters_text.py
from __future__ import annotations
from .text_steps import TextSteps

class TextStripStep:
    def apply(self, s: str) -> str:
        return TextSteps.strip(s)

class TextNormalizeStep:
    def apply(self, s: str) -> str:
        return TextSteps.normalize_whitespace(s)

File: src/typescript/examples/adapters_text.ts
import { TextSteps } from "./text_steps.js";

export class TextStripStep {
  apply(value: string): string {
    return TextSteps.strip(value);
  }
}

export class TextNormalizeStep {
  apply(value: string): string {
    return TextSteps.normalizeWhitespace(value);
  }
}
